docs/docfx.json

```diff
@@ -16,63 +38,25 @@
         ]
       }
     ],
-    "dest": "_site",
-    "disableGitFeatures": false,
-    "fileMetadataFiles": [],
-    "globalMetadata": {
-      "_appFaviconPath": "images/favicon.ico",
-      "_appFooter": "DSharpPlus.CommandAll",
-      "_appLogoPath": "images/logo.png",
-      "_appName": "CommandAll",
-      "_appTitle": "DSharpPlus.CommandAll"
-    },
-    "globalMetadataFiles": [],
-    "keepFileLink": false,
-    "noLangKeyword": false,
-    "overwrite": [
-      {
-        "exclude": [
-          "obj/**",
-          "_site/**"
-        ],
-        "files": [
-          "apidoc/**.md"
-        ]
-      }
-    ],
-    "postProcessors": [],
     "resource": [
       {
         "files": [
           "images/**"
         ]
       }
     ],
-    "template": [
-      "default",
-      "../libs/SingulinkFX/singulinkfx"
-    ],
     "xref": [
       "https://dsharpplus.github.io/DSharpPlus/xrefmap.yml"
     ],
-    "xrefService": "https://xref.docs.microsoft.com/query?uid={uid}"
-  },
-  "metadata": [
-    {
-      "dest": "api",
-      "disableDefaultFilter": false,
-      "disableGitFeatures": false,
-      "includePrivateMembers": false,
-      "namespaceLayout": "flattened",
-      "noRestore": false,
-      "src": [
-        {
-          "files": [
-            "*.csproj"
-          ],
-          "src": "../src"
-        }
-      ]
-    }
-  ]
-}
\ No newline at end of file
+    "output": "_site",
+    "globalMetadataFiles": [],
+    "fileMetadataFiles": [],
+    "template": [
+      "default",
+      "modern"
+    ],
+    "postProcessors": [],
+    "keepFileLink": false,
+    "disableGitFeatures": false
```

Comment:
How many properties here contain their default values?

Reply:
all of them, except xref. that's set so that links to d#+ entities work right

Reply:
So in theory, by removing all properties except `xref`, the docfx will have no difference?

Reply:
.....don't try it, it could just break entirely

Reply:
I'm gonna try it